src/hub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -647,26 +651,49 @@ def setUp(self) -> None:
         ) as fh:
             self.subscription_reactivate_event = json.loads(fh.read())
 
+        with open(
+            "src/hub/tests/unit/fixtures/stripe_sub_updated_event_change.json"
+        ) as fh:
+            self.subscription_change_event = json.loads(fh.read())
+
         with open(
             "src/hub/tests/unit/fixtures/stripe_sub_updated_event_no_trigger.json"
         ) as fh:
             self.subscription_updated_event_no_match = json.loads(fh.read())
 
+        with open("src/hub/tests/unit/fixtures/stripe_previous_plan1.json") as fh:
```

Comment:
save the dir as `src/hub/tests/unit/fixtures` instead of copy and pasting it over and over a thousand times; this will also make the `with open` expressions shorter and not have `black` do the ugly wrap

Reply:
done